homeworks/maksym.selenkov_MaksymSelenkov/5-rxjs/app.js

```diff
@@ -0,0 +1,36 @@
+const { fromEvent } = window.rxjs;
+const {
+    map, pairwise, throttleTime, filter,
+} = window.rxjs.operators;
+
+const headerEl = document.querySelector('[data-header]');
+const burgerEl = document.querySelector('[data-burger]');
+const navigationEl = document.querySelector('[data-navigation]');
+const buyEl = document.querySelector('[data-buy]');
+
+fromEvent(window, 'scroll')
+    .pipe(
+        throttleTime(150),
+        map((e) => e.currentTarget.scrollY),
+        pairwise(),
+        filter((v) => Math.abs(v[1] - v[0]) > 50),
```

Comment:
[non-blocking] For me, it's better to destruct arguments and give them descriptive names. `previousScroll` and `currentScroll` sounds better, then `v[1]` and `v[0]`